Logger.Tests/BaseLoggerMixinsTests.cs

```diff
@@ -26,14 +62,59 @@ public void Error_WithData_LogsMessage()
             var logger = new TestLogger();
 
             // Act
-            //logger.Error("Message {0}", 42);
+            logger.Error("Message {0}", 42);
 
             // Assert
             Assert.AreEqual(1, logger.LoggedMessages.Count);
             Assert.AreEqual(LogLevel.Error, logger.LoggedMessages[0].LogLevel);
             Assert.AreEqual("Message 42", logger.LoggedMessages[0].Message);
         }
 
+        [TestMethod]
+        public void Warning_WithData_LogsMessage()
```

Comment:
This works, but many of these tests could be combined into a single data driven test using `DataRow`. Like this: https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.datarowattribute?view=mstest-net-1.3.2#examples